site/content/en/docs/admission-check-controllers/provisioning.md

```diff
@@ -10,6 +10,8 @@ The Provisioning Admission Check Controller is an Admission Check Controller des
 
 The controller is part of kueue. You can enable it by setting the `ProvisioningACC` feature gate. Check the [Installation](/docs/installation/#change-the-feature-gates-configuration) guide for details on feature gate configuration.
 
+The Provisioning Admission Check Controller is supported on [Kubernetes cluster-autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) versions 1.29 and later.
```

Comment:
Do we have some reference to support this is 1.29+?

/cc @kisieland

Reply:
I was not able to find anything in the release notes of CA 1.29. However, I see the below PR was merged on Jan 3rd 2024. The first new CA release tag that was cut after Jan 3rd was 1.29.0 
https://github.com/kubernetes/autoscaler/pull/6386

Also, verified that the ProvisiongRequest directory is only present in CA 1.29.0 tag and not in the previous (1.28.2) or earlier tags.

Reply:
@kisieland to confirm

Reply:
@kisieland gentle reminder to take a look at this PR.

Reply:
Hi, sorry for the late response. I had a lot of work over last week.
The Provisioning Request CRD support in CA should be implemented in 1.30+
There are supposed to be two more PRs. 
The PR that is already implemented: https://github.com/kubernetes/autoscaler/pull/6451
There is one in review: https://github.com/kubernetes/autoscaler/pull/6488

Reply:
Although some providers, like GKE, can have an implementation, but no earlier than 1.29

Reply:
Thanks for the responses @kisieland @alculquicondor . I'll keep the PR open and update it once the feature is completely available.
Let me know if the PR should be closed instead.

Reply:
maybe you can say something along the lines of:
The ProvisioningRequest was added in cluster-autoscaler v1.29, but some cloud providers might not have an implementation for it.

Reply:
I have updated the text. Please take a look if it looks good now.